src/test/java/com/fundation/webservice/model/CriteriaKeyFrameVideoTest.java

```diff
@@ -0,0 +1,33 @@
+package com.fundation.webservice.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
```

Comment:
Please insert a description for this test class.

Reply:
Okay, it's fixed.